<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Factura</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAct"/>
   <addaction name="actionAbrir"/>
   <addaction name="saveAct"/>
   <addaction name="separator"/>
   <addaction name="previewAct"/>
   <addaction name="printAct"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="newAct"/>
    <addaction name="actionAbrir"/>
    <addaction name="saveAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="menuProductos">
    <property name="title">
     <string>Productos</string>
    </property>
    <addaction name="actionEditar_Productos"/>
    <addaction name="actionInventario"/>
   </widget>
   <widget class="QMenu" name="menuVentanas">
    <property name="title">
     <string>Ventanas</string>
    </property>
    <addaction name="closeAct"/>
    <addaction name="closeAllAct"/>
    <addaction name="separator"/>
    <addaction name="tileAct"/>
    <addaction name="cascadeAct"/>
    <addaction name="separator"/>
    <addaction name="nextAct"/>
    <addaction name="previousAct"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="aboutAct"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuProductos"/>
   <addaction name="menuVentanas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionAnular">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/edit-delete.png</normaloff>:/icons/res/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>anular</string>
   </property>
   <property name="toolTip">
    <string>Anular la Factura</string>
   </property>
  </action>
  <action name="actionRecibo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/okteta.png</normaloff>:/icons/res/okteta.png</iconset>
   </property>
   <property name="text">
    <string>Recibo</string>
   </property>
   <property name="toolTip">
    <string>Recibos</string>
   </property>
  </action>
  <action name="newAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-new.png</normaloff>:/icons/res/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Factura</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-edit.png</normaloff>:/icons/res/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-save.png</normaloff>:/icons/res/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionEditar_Productos">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/view-bank-account.png</normaloff>:/icons/res/view-bank-account.png</iconset>
   </property>
   <property name="text">
    <string>Editar Productos</string>
   </property>
  </action>
  <action name="closeAct">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="closeAllAct">
   <property name="text">
    <string>Cerrar Todo</string>
   </property>
  </action>
  <action name="tileAct">
   <property name="text">
    <string>Titulo</string>
   </property>
  </action>
  <action name="cascadeAct">
   <property name="text">
    <string>Cascada</string>
   </property>
  </action>
  <action name="nextAct">
   <property name="text">
    <string>Siguiente</string>
   </property>
  </action>
  <action name="previousAct">
   <property name="text">
    <string>Anterior</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>Acerda de ...</string>
   </property>
  </action>
  <action name="printAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-print.png</normaloff>:/icons/res/document-print.png</iconset>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="previewAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-preview.png</normaloff>:/icons/res/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Vista Previa</string>
   </property>
  </action>
  <action name="actionInventario">
   <property name="text">
    <string>Inventario</string>
   </property>
  </action>
  <action name="fontAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/font.png</normaloff>:/icons/res/font.png</iconset>
   </property>
   <property name="text">
    <string>Tipo de Letra</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
